lib/apivore/rspec_builder.rb

```diff
@@ -32,7 +32,7 @@ def apivore_build_path(path, data)
           raise URI::InvalidURIError, "No substitution data found for {#{key}} to test the path #{path}.\nAdd it via an:\n  apivore_setup '<path>', '<method>', '<response>' do\n    { '#{key}' => <value> }\n  end\nblock in your specs.", caller
         end
       end
-      path
+      path + (data['_query_string'] ? "?#{data['_query_string']}" : '')
```

Comment:
@gwshaw Yeah, I've been digging further, something else is going on. For some reason the 200 get response test 'data' is coming through as `#<RSpec::Mocks::MessageExpectation:0x007ffb8a24ed38>` instead of a Hash, even when the tests were passing. That's just wrong, I'm trying to work it out and fix. It looks like there is possibly a hidden bug which your change has somehow surfaced. Leave it with me and I'll try to fix it and get your feature in.